lib/index.ts

```diff
@@ -48,13 +66,13 @@ export async function inspect(
   }
 }
 
-export function buildArgs(root, targetFile, mavenArgs) {
+export function buildArgs(
+  targetFile?: string,
+  mavenArgs?: string[] | undefined,
```

Comment:
doesn't the `?` imply `undefined`?

Reply:
yes I think so, but it's the definition of options.arg in cli-interface that defines this